lib/matplotlib/tests/test_figure.py

```diff
@@ -287,6 +287,16 @@ def test_suptitle_fontproperties():
     assert txt.get_weight() == fps.get_weight()
 
 
+@image_comparison(['figure_suptitle_subfigures'],
+                  extensions=['png'])
```

Comment:
```suggestion
                  extensions=['png'], style='mpl20')
```
Not compulsory, but the [newer style is preferred](https://matplotlib.org/devdocs/devel/testing.html#writing-an-image-comparison-test).  If you decide to change this, the image will also need replacing.  Adding and replacing an image in separate commits in the same PR will cause a CI failure, so you would also need to [squash the commits down to one](https://matplotlib.org/devdocs/devel/development_workflow.html#rewriting-commit-history).  So whether you take this suggestion should depend on how confident you feel with git!

Reply:
Thanks @rcomer, I made the changes you suggested. I also squashed down the commits to avoid the image change issue in the CI pipeline.

Reply:
Well, looks like the added-modified check on that image failed. I'll investigate and get it sorted.